sentry/src/main/java/io/sentry/JsonObjectSerializer.java

```diff
@@ -46,6 +46,8 @@ public void serialize(
       serializeCollection(writer, logger, Arrays.asList((Object[]) object));
     } else if (object instanceof Map) {
       serializeMap(writer, logger, (Map<?, ?>) object);
+    } else if (object.getClass().isEnum()) {
+      writer.value(object.toString());
```

Comment:
WHat's about `object..name()`? Does it return the same thing?

Reply:
We'd have to use reflection to invoke `.name()`. By default `.toString()` should be the same as `.name()`. It also allows overriding the value that is serialized.